packages/actor-query-operation-group/examples/translate_to_algebra.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const fs = require('fs');
+const { translate } = require('sparqlalgebrajs');
+
+function main() {
```

Comment:
Like above, PR'ing this to sparqlalgebrajs may make more sense, as this tool is not exclusively useful for the GROUP actor :-)